deploy/LiveDeploy.js

```diff
@@ -102,13 +102,6 @@ module.exports = async function (hre) {
         contract: "GasOracle",
     })
 
-    const gasPriceOracleAdapter = await deploy("GasPriceOracleAdapter", {
```

Comment:
Just checking, we don't need this for the gas oracle? I believe that txn.gasPrice is reported in wei and the chainlink oracle reports gwei so we may need to convert units

Reply:
Disregard. Saw your comment about it being a wrapper on a wrapper and that is correct